libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/overlay/TapToRouteOverlay.kt

```diff
@@ -258,9 +282,9 @@ class TapToRouteOverlay(private val mapView: CycleMapView) : Overlay(), TapListe
     ////////////////////////////////////
     private enum class TapToRoute private constructor(val waypointingInProgress: Boolean, val actionDescription: String) {
         WAITING_FOR_START(false, "Tap map to set Start"),
-        WAITING_FOR_SECOND(true, "Tap map to set Waypoint"),
+        WAITING_FOR_SECOND(true, "Tap map to set Waypoint\nTap here for Circular Route"),
```

Comment:
I would propose changing this string to: `Tap map to set Waypoint, or\nTap here for Circular Route`

Aside: we should probably move these texts into the languages XML so they can be internationalised

Reply:
I guess I should do the same for "Tap map to set Waypoint\nTap here to Route" --> "Tap map to set Waypoint, or\nTap here to Route".

Yes, good point, I'll move the strings.  I'm trying to move them whenever I come across them, but I was probably too bogged down in other stuff at the time and then forgot! ( - and forgot to put in a //todo)

Reply:
I couldn't work out how to access the context easily from inside the enum class, so have done the context.getString outside the class.  From what I can see, the actionDesc for ALL_DONE doesn't get called, but I've done a check just in case.